demo/index.css

```diff
@@ -1,46 +1,579 @@
-﻿[draggable] {
-  user-select: none;
+﻿* {
```

Comment:
Why are we adding a `*` rule here? Just assign the `font-family` for the `html,body` rule, and the C in CSS will take care of the rest.